### ISSUE TITLE
Training code, reproduce the accuracy of the paper

### ISSUE THREAD

**Issue by pytho12** [z-x-yang/CFBI#51]
Hello, I have trained your code, but there is a 1.2% difference in the accuracy of both DAVIS16 and DAVIS17 datasets. Are there any tips that can help improve the code accuracy?

**Comment by pytho12** [comment]
我在训练YouTube和DAVIS2017时，batch_size都设置为4，您论文中训练DAVIS2017时的学习率是0.006，但是您的代码里却是0.01，应该使用哪个学习率为准呢

**Comment by pytho12** [comment]
When I trained YouTube and DAVIS2017, batch_size was set to 4. The learning rate of training DAVIS2017 in your paper is 0.006, but it is 0.01 in your code. Which learning rate should be used

**Comment by z-x-yang** [comment]
Try half LR, double training steps.

**Comment by pytho12** [comment]
Are you talking about doubling the epochs of both the training YouTube and DAVIS processes? I see your code is not adaptive to adjust the learning rate, will it be effective?

**Comment by z-x-yang** [comment]
It's a common practice when using half of the batch size. Not sure if this will be effective. The best way for reproducing is to use the default DAVIS setting of this repository.
